resources/components/evidence-modal.vue

```diff
@@ -58,13 +66,24 @@ export default {
     },
     isPdf () {
       return ['.pdf'].includes(this.returnExtension())
+    },
+    isVideo () {
+      return ['.mp4', '.ogg', '.webm'].includes(this.returnExtension())
     }
   },
 
   methods: {
     onClose () {
       this.$emit('close')
     },
+    getVideoMIME () {
```

Comment:
I think you should convert this into a `computed` property as it feels more semantic and relates to Vue reactive behavior instead of running a function each render cycle.

If I may say, `videoMIME` would be also better naming on the computed property. 

The implementation is great btw!

Reply:
OK, thanks, will do in a minute.